src/plugins/vis_type_table/public/components/table_vis_component_group.tsx

```diff
@@ -7,8 +7,9 @@ import React, { memo } from 'react';
 
 import { IInterpreterRenderHandlers } from 'src/plugins/expressions';
 import { TableGroup } from '../table_vis_response_handler';
-import { TableVisConfig, TableUiState } from '../types';
+import { TableVisConfig } from '../types';
 import { TableVisComponent } from './table_vis_component';
+import { TableUiState } from '../utils/get_table_ui_state';
```

Comment:
nit - this should be re-exported if necessary and imported from

```suggestion
import { TableUiState } from '../utils';
```

Reply:
will revise